packages/office-ui-fabric-react/src/components/Link/Link.tsx

```diff
@@ -41,10 +42,12 @@ export class Link extends BaseComponent<ILinkProps, any> implements ILink {
               'ms-Link',
               styles.root,
               className,
-              disabled && ('is-disabled ' + styles.isDisabled)
+              disabled && ('is-disabled ' + styles.isDisabled),
```

Comment:
if "styles.isDisabled" is undefined... what happens?

Reply:
just trying to figure out why the snapshot has "undefined" in the classname.

Reply:
per offline discussion, I'll leave this as is because it's a nuance of how the jest config treats scss